Add a note about setting the value of maxNumConcurrentSubTasks (#12772)
apache/druid: docs/ingestion/native-batch-input-source.md

* Similar to file-based input formats, any updates to existing data will replace the data in segments specific to the intervals specified in the `granularitySpec`.

## Combining input source

The Combining input source lets you read data from multiple input sources.
It identifies the splits from delegate input sources and uses a worker task to process each split.
Use the Combining input source only if all the delegates are _splittable_ and can be used by the [Parallel task](./native-batch.md). 

Similar to other input sources, the Combining input source supports a single `inputFormat`.
Delegate input sources that require an `inputFormat` must have the same format for input data.

|Property|Description|Required|
|--------|-----------|---------|

> |type|This should be "combining".|Yes|

Author: Excellent suggestion!

> |delegates|List of _splittable_ input sources to read data from.|Yes|

Reviewer: ```suggestion
|delegates|List of splittable input sources to read data from.|Yes|
```

**Recommended practices**

> Compared to the other native batch input sources, SQL input source behaves differently in terms of reading the input data and so it would be helpful to consider the following points before using this input source in a production environment:

Reviewer 2: ```suggestion
Compared to the other native batch input sources, SQL input source behaves differently in terms of reading the input data. Therefore, consider the following points before using this input source in a production environment:
```

...
```

> For the Combining input source to read data correctly, set the value of `maxNumConcurrentSubTasks` in `tuningConfig` as follows:

Reviewer: Adds a bit more context
```suggestion
The [secondary partitioning method](native-batch.md#partitionsspec) determines the requisite number of concurrent worker tasks that run in parallel to complete ingestion with the Combining input source.
Set this value in `maxNumConcurrentSubTasks` in `tuningConfig` based on the secondary partitioning method:
```
